README.md

```diff
@@ -4,28 +4,27 @@
 
 ## Scope
-* Service APIs for “IdentityAndConsentManagement” (see [APIBacklog.md](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/APIBacklog.md))  
+* Service APIs for “IdentityAndConsentManagement” (see [APIBacklog.md](https://github.com/camaraproject/APIBacklog/blob/main/documentation/APIbacklog.md))  
 * Telco operator exposure platforms implementing CAMARA APIs should be built with a privacy-by-default approach to fully comply with data protection regulations, such as the GDPR regulation in Europe, which emphasises on user privacy. These regulations note that some CAMARA APIs may require user consent to be accessed. This forces the operators to provide means and appropriate solutions to capture, store and manage this consent through its lifecycle. Otherwise, the scoped CAMARA APIs cannot be rolled out in production networks. Building such a solution also means bringing in scope the identity of the end user and/or the subscriber (as both could be different) and making sure that end user experience of using the API is not compromised while doing so.
 * Describe, develop, document and test the APIs (with 1-2 Telcos)  
 * Started: March 2023
-* Location: virtually
 
-## Meetings
-* Meetings are held virtually
-* Schedule: bi-weekly, Wednesday, 4 PM CET/CEST (15:00 UTC, 14:00 UTC during European DST). For next call, see minutes of [previous meetings](documentation/MeetingMinutes) 
-* Meetings are held virtually: [Registration / Join](https://zoom-lfx.platform.linuxfoundation.org/meeting/94629188836?password=278b4c8a-f370-43bf-bac1-b30a39f169f3) 
+## Release Information
 
-## Status and released versions
-* Release 0.1.0 of guidelines and documentation for CAMARA APIs is available within the [release-0.1.0 branch](https://github.com/camaraproject/IdentityAndConsentManagement/tree/release-0.1.0)
-
-For changes see [CHANGELOG.md](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/CHANGELOG.md)
-
-## Contributorship and mailing list
-* To subscribe / unsubscribe to the mailing list of this Sub Project and thus be / resign as Contributor please visit <https://lists.camaraproject.org/g/sp-icm>.
-* A message to all Contributors of this Sub Project can be sent using <sp-icm@lists.camaraproject.org>.
+* The latest public release of guidelines and documentation for CAMARA APIs is available [here](https://github.com/camaraproject/IdentityAndConsentManagement/releases/latest).
+* For previous releases, pre-releases and changes see [CHANGELOG.md](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/CHANGELOG.md)
 
+## Contributing
+* Meetings
+  * Bi-weekly Wednesday, 16:00 CEST (14:00 UTC during European daylight saving time)
+  * [Registration / Join](https://zoom-lfx.platform.linuxfoundation.org/meeting/94629188836?password=278b4c8a-f370-43bf-bac1-b30a39f169f3)
+  * Minutes: Access [meeting minutes](https://wiki.camaraproject.org/x/2QBFAQ).
```

Comment:
```suggestion
* Meetings
  * [Registration](https://zoom-lfx.platform.linuxfoundation.org/meeting/94629188836?password=278b4c8a-f370-43bf-bac1-b30a39f169f3)
   * Schedule: Bi-weekly Wednesday, 16:00 CEST (14:00 UTC during European daylight saving time)
  * Minutes: Access [meeting minutes](https://wiki.camaraproject.org/x/2QBFAQ).
```

Reply:
Agreed, just that I added the "/ Join" again as you can't use the link for registration as long the meeting is active. Will change the template accordingly.